plugins/airtable/src/ui.ts

```diff
@@ -0,0 +1,33 @@
+import { framer } from "framer-plugin"
+
+export async function showNoTableAccessUI() {
+    framer.showUI({
+        width: 240,
+        height: 110,
+        resizable: false,
+    })
+}
+
+export async function showFieldMappingUI() {
+    framer.showUI({
+        width: 425,
+        height: 425,
+        minWidth: 360,
+        minHeight: 425,
+        resizable: true,
+    })
+}
+
+export async function showDataSourceSelectionUI() {
+    framer.showUI({
+        width: 320,
+        height: 345,
+    })
+}
+
+export async function showLoginUI() {
+    framer.showUI({
+        width: 320,
+        height: 340,
+    })
+}
```

Comment:
If it doesn't reset `resizable` to `false`, the window keeps its size when going back to the login UI but only if you've ever resized the plugin before. 

Adding `resizable: false` fixes this.

<img width="522" height="505" alt="image" src="https://github.com/user-attachments/assets/31ee8dab-5450-456a-8931-2ae031b65e82" />


```suggestion
export async function showDataSourceSelectionUI() {
    framer.showUI({
        width: 320,
        height: 345,
        resizable: false,
    })
}

export async function showLoginUI() {
    framer.showUI({
        width: 320,
        height: 340,
        resizable: false,
    })
}
```

Reply:
Nice one - thanks